### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by j616**
# Details
Reduce size of video grains generated by hypothesis in tests. This addresses an issue where hypothesis now restricts the size of test data which it generates. [Large sizes never actually worked correctly](https://github.com/HypothesisWorks/hypothesis/issues/3896).

# Pivotal Story
Story URL:

# Related PRs
_Where appropriate. Indicate order to be merged._

# Links to external test runs/working deployment
_Where appropriate, if separate to default CI run_

# Submitter PR Checks
_(tick as appropriate)_

- [ ] PR completes task/fixes bug
- [ ] Tests exercise code appropriately
- [ ] New features and API breaks are flagged in commit messages using magic strings
- [ ] Documentation updated (README, Confluence, Docstrings, API spec, Engineering Guide, etc.)
- [ ] Downstream repos have been checked for potential breaks & fixed as needed
- [ ] APIs/UIs/CLIs updated as required
- [ ] PR added to Pivotal story
- [ ] Follow-up stories added to Pivotal
- [ ] Any temporary code/configuration removed (e.g. test deployment environment, temporary commontooling branch)
- [ ] Any pins against pre-releases have been removed

# Reviewer PR Checks
_(tick as appropriate)_

- [ ] PR completes task/fixes bug
- [ ] Tests exercise code appropriately
- [ ] Design makes sense, and fits with our current code base
- [ ] Code is easy to follow
- [ ] PR size is sensible
- [ ] Commit history is sensible and tidy

# Info on Cloudfit PRs
- The checks above are guidelines. They don't all have to be ticked, but they should all have been considered.
- For more details on how to asses PRs, see: https://github.com/bbc/rd-apmm-docs-ways-of-working/blob/master/workflow/PRChecklist.md

### CODE REVIEW

File: mediagrains/hypothesis/strategies.py [hunk 1]
```diff
@@ -483,7 +483,11 @@ def grains_with_data(grain_type):
     """Strategy giving grains which have data payloads filled out using an appropriate strategy for the grain type.
 
     :param grain_type: The type of grains to generate"""
-    if grain_type in ("audio", "video", "coded_audio", "coded_video"):
+    if grain_type is "video":
+        return grains(grain_type, width=16, height=9).flatmap(lambda g: grains_from_template_with_data(g))
+    elif grain_type is "coded_video":
+        return grains(grain_type, coded_width=16, coded_height=9).flatmap(lambda g: grains_from_template_with_data(g))
+    elif grain_type in ("audio", "video"):
```
Comment (philipnbbc) [inline mediagrains/hypothesis/strategies.py:490]: Should be audio and coded audio?
```suggestion
    elif grain_type in ("audio", "coded_audio"):
```

File: mediagrains/hypothesis/strategies.py [hunk 2]
```diff
@@ -483,7 +483,11 @@ def grains_with_data(grain_type):
     """Strategy giving grains which have data payloads filled out using an appropriate strategy for the grain type.
 
     :param grain_type: The type of grains to generate"""
-    if grain_type in ("audio", "video", "coded_audio", "coded_video"):
+    if grain_type == "video":
+        return grains(grain_type, width=16, height=9).flatmap(lambda g: grains_from_template_with_data(g))
+    elif grain_type == "coded_video":
+        return grains(grain_type, coded_width=16, coded_height=9).flatmap(lambda g: grains_from_template_with_data(g))
```
Comment (philipnbbc) [inline mediagrains/hypothesis/strategies.py:489]: Nit: Use origin_width/height instead or not set anything at all. We don't use coded_width/height. E.g. origin_width/height are mapped to the Flow's frame_width/height.

**Review by philipnbbc — APPROVED** [review]
Some changes suggested inline. Otherwise LGTM

**Comment by philipnbbc** [comment]
Oh, it was changed to draft as I reviewed it :-)

**Comment by j616** [comment]
Fixes made anyway :)

**Pull request merged**